src/storage/storage.repository.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { CreateStorageBodyDto } from './dto';
+
+@Injectable()
+export class StorageRepository {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  async create(data: CreateStorageBodyDto) {
```

Comment:
Usually repository should have its own data params. And we map dtos to that data param. But I think this is better since, we need to write code faster and also, we will be adding entities to our project. In future, repository will await entity as param